apps/src/lab2/views/components/ResizeBar.tsx

```diff
@@ -53,21 +59,31 @@ const ResizeBar: React.FunctionComponent<ResizeBarProps> = ({
     ? moduleStyles.verticalBar
     : moduleStyles.horizontalBar;
 
+  const {onPointerDown, ...mainSeparatorProps} = separatorProps;
+
   return (
     <div className={classNames(moduleStyles.resizeBar, layoutClass)}>
+      {/* The visible bar */}
       <div
-        className={classNames(moduleStyles.grabbableDiv, grabbableClass)}
-        {...separatorProps}
+        className={classNames(moduleStyles.absoluteBar, resizingBarClass)}
+        {...mainSeparatorProps}
         // TODO: the separator props are applying role "separator" as well as min/max/now aria values.
         // Is it ok to ignore this warning?
         // https://developer.mozilla.org/en-US/docs/Web/Accessibility/ARIA/Roles/separator_role
         // eslint-disable-next-line jsx-a11y/no-noninteractive-tabindex
         tabIndex={0}
-        onFocus={() => setIsActive(true)}
-        onBlur={() => setIsActive(false)}
-        onMouseEnter={() => setIsActive(true)}
-        onMouseLeave={() => setIsActive(false)}
-      />
+      >
```

Comment:
Seems like the resize bar is still keyboard navigable but no longer highlighted.

On prod:

https://github.com/user-attachments/assets/5ec260fc-c454-486b-8c8e-707bdacd2915


On your branch:

https://github.com/user-attachments/assets/06cffd15-172d-4f28-9df4-a8a4c84f6e0a

Do we need to update the `tabIndex`?

Reply:
Huh it was highlighting for me when I tested it, I'll check again on different browsers.

Reply:
Fixed! I needed to set `isActive` to `true` on focus.

Reply:
🎉